VAMobile/src/utils/claims.tsx

```diff
@@ -262,11 +262,11 @@ export const deletePhoto = (deleteCallbackIfUri: (response: Asset[]) => void, de
  * @param fs - fontscale function
  */
 export const getIndicatorCommonProps = (fs: (val: number) => number) => {
-  const indicatorDiameter = 30
+  const indicatorDiameter = fs(30)
   return {
-    height: fs(indicatorDiameter),
-    width: fs(indicatorDiameter),
-    borderRadius: fs(indicatorDiameter),
+    height: indicatorDiameter > 35 ? 35 : indicatorDiameter,
```

Comment:
Couldn't the entire logic be pulled into the const since it's the same for all 3?